### PR TITLE
refs #257: Allowing defining the remote endpoint in #sync().

### PR DESCRIPTION
**Pull request by n1k0**
Took the easiest way to allow passing the remote to `#sync()`. Works, but ugly, as I'm mutating the api instance during the time of the sync flow execution :/ That's the price to pay for strict BC.

Later we'll move the whole sync flow to its own module, so that's a temporary situation (famous last words).

r=? @leplatrem

### CODE REVIEW

File: src/collection.js [hunk 1]
```diff
@@ -764,18 +764,31 @@ export default class Collection {
...
-  sync(options={strategy: Collection.strategy.MANUAL, headers: {}, ignoreBackoff: false}) {
+  sync(options={
+    strategy: Collection.strategy.MANUAL,
+    headers: {},
+    ignoreBackoff: false,
+    remote: null,
+  }) {
+    const previousRemote = this.api.remote;
+    if (options.remote) {
+      console.log("using custom remote", options.remote);
```
Comment (leplatrem) [inline src/collection.js]: Add a bit of context like `#sync()` prefix ?

**Comment by leplatrem** [comment]
It looks good to me but I'm sorry, I didn't take the time to review it seriously. Be back next week!

**Comment by leplatrem** [comment]
r+ :)

File: src/collection.js (hunk 1 above)
Comment (michielbdejong) [inline src/collection.js]: Should kinto.js contain console.log statements? Just noting that this seems to be the first/only console.log statement so far.

Comment (n1k0) [inline src/collection.js]: No it shouldn't. It was probably a debug statement :) I'll drop it.

File: docs/tutorial.md [hunk 2]
```diff
@@ -76,7 +63,7 @@ We want to listen to form submission events to add tasks into our local database
 
 ```js
 function main() {
-  var db = new Kinto({remote: "https://kinto.dev.mozaws.net/v1"});
```
Comment (almet) [inline docs/tutorial.md]: It's still possible, right?

Comment (n1k0) [inline docs/tutorial.md]: Yeah but no more encouraged

**Pull request merged**